llvm/unittests/ExecutionEngine/Orc/OrcCAPITest.cpp

```diff
@@ -90,6 +90,35 @@ class OrcCAPITestBase : public testing::Test {
 
     LLVMOrcDisposeLLJIT(J);
     TargetSupported = true;
+
+    // Create test functions in text format, with the proper extension
+    // attributes.
+    if (SumExample.empty()) {
+      std::string I32RetExt = "";
+      std::string I32ArgExt = "";
+      if (StringRef(TargetTriple).starts_with("s390x-ibm-linux"))
+        I32RetExt = I32ArgExt = "signext ";
+
+      std::ostringstream OS;
+      OS << "define " << I32RetExt << " i32 "
+         << R"(@sum()" << "i32 " << I32ArgExt << "%x, i32 " << I32ArgExt << "%y)"
```

Comment:
I think `R"(@sum()"` could just be `"@sum("`.

Reply:
yep